src/components/Profile.js

```diff
@@ -96,6 +90,15 @@ function Profile() {
     setShowSection(MY_SCHEDULE);
   };
 
+  function logOut() {
+    signOut(auth)
+      .then(() => {
+        props.setLogin(false);
+        localStorage.clear();
+      })
+      .catch((error) => {});
```

Comment:
at least log here... You catch and do nothing will cause undebuggable error in future...